pkg/k8s/event_handlers.go

```diff
@@ -16,9 +16,9 @@ type observeFilter func(obj interface{}) bool
 
 // EventTypes is a struct helping pass the correct types to GetKubernetesEventHandlers
 type EventTypes struct {
-	Add    a.AnnouncementType
-	Update a.AnnouncementType
-	Delete a.AnnouncementType
+	Add    announcements.AnnouncementType
```

Comment:
(an idea to improve readabality)
It might be prudent to 
1. rename `announcements` to `announcement` (drop the plural) and 
2. rename `AnnouncementType` to `Type`

so then this would read `announcement.Type`